test/test_dist_autograd.py

```diff
@@ -81,6 +95,38 @@ def test_autograd_send_function(self):
             self.assertEqual(t2, next_funcs[1][0].variable)
             self.assertEqual(0, next_funcs[1][1])
 
+            # Test recv functions.
+            recv_functions = ctx._recv_functions()
+            self.assertEqual(1, len(recv_functions))
+            self.assertEqual(ret.grad_fn, list(recv_functions.values())[0])
+
+            # We should have send/recv functions from the previous rank, get all
+            # contexts in this node to find them.
+
+            # Wait for the prev rank to be done with rpc.
+            while not prev_rank_rpc_done:
+                pass
```

Comment:
Do we need to sleep a few milliseconds in each iteration to just yield? When working together with `py::object` RRef, it needs to acquire the GIL to unpickle `PyRRef` objects. Will this be a problem if we busy looping here?